src/tfvc/tfvcscmprovider.ts

```diff
@@ -75,6 +79,39 @@ export class TfvcSCMProvider implements SCMProvider {
         }
     }
 
+    public static async Exclude(path: string): Promise<void> {
+        const tfvcProvider: TfvcSCMProvider = TfvcSCMProvider.instance;
+        if (!tfvcProvider) {
+            // We are not the active provider
+            Logger.LogDebug("Failed to GetCheckinInfo. TFVC is not the active provider.");
+            throw TfvcError.CreateInvalidStateError();
+        }
+
+        await tfvcProvider._model.Exclude(path);
+    };
+
+    public static async Refresh(): Promise<void> {
+        const tfvcProvider: TfvcSCMProvider = TfvcSCMProvider.instance;
```

Comment:
looks like we are reusing this invalid state check a lot. Consider refactoring it into it's own method that gets the scm provider.